modules/openshift-cluster-maximums-environment.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc
+
+[id="cluster-maximums-environment_{context}"]
+= {product-title} environment and configuration on which the cluster maximums are tested
+
+AWS cloud platform:
+
+[options="header",cols="8*"]
+|===
+| Node |Flavor |vCPU |RAM(GiB) |Disk type|Disk size(GiB)/IOS |Count |Region
+
+| Master/Etcd footnoteref:[masteretcdnode, io1 disks with 3000 IOPS are used for master/etcd nodes as etcd is i/o intensive and latency sensitive.]
+| r5.4xlarge
+| 16
+| 128
+| io1 
+| 220 / 3000
+| 3
+| us-west-2
+
+| Infra footnoteref:[infranodes,Infra nodes are used to host openshift-monitoring, ingress and router components to make sure they have enough resources to run at large scale. Procedure on creating Infra machinesets can be found at https://docs.openshift.com/container-platform/4.3/machine_management/creating-infrastructure-machinesets.html.]
+| m5.12xlarge
+| 48
+| 192
+| gp2 
+| 100 
+| 3
+| us-west-2
+
+| Workload footnoteref:[workloadnode, Workload node is dedicated to run performance/scalability workload generators.]
+| m5.4xlarge
+| 16
+| 64
+| gp2 
+| 500 footnoteref:[disksize, Larger disk size is used to have enough space to store large amounts of data collected during the performamce/scalability test run.]
```

Comment:
performance and scalability 
(typo in "performance" as well)

Reply:
Use a larger disk size so that there is enough space to store the large amounts of data that is collected during the performance and scalability test run.